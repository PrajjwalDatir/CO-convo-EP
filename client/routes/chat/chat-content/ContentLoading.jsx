import React from 'react';
import ContentContainer from './ContentContainer';
import styles from './content-loading.module.css';

function ContentLoading() {
  return (
    <ContentContainer>
      <div className="flex flex-grow justify-center items-center">
        <h1 className="text-2xl text-gray-700">Loading...</h1>
        <div
          className={`w-48 h-48 flex justify-center items-center ${
            styles['spinner-parent']
          }`}
        >
          <div
            className={`rounded-full w-20 h-20 bg-blue-200 ${styles.spinner}`}
          />
        </div>
      </div>
    </ContentContainer>
  );
}

export default ContentLoading;
