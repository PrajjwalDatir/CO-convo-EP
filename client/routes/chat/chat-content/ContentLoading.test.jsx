import React from 'react';
import { mount } from 'enzyme';
import ContentLoading from './ContentLoading';

describe('<ContentLoading />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<ContentLoading />);
    expect(wrapper).toMatchSnapshot();
  });
});
