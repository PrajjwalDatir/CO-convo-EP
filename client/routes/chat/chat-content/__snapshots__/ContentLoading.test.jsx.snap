// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentLoading /> should match snapshot 1`] = `
<ContentLoading>
  <ContentContainer
    direction="flex-row"
  >
    <div
      className="my-8 flex flex-grow bg-white rounded-lg shadow-xl flex-row"
    >
      <div
        className="flex flex-grow justify-center items-center"
      >
        <h1
          className="text-2xl text-gray-700"
        >
          Loading...
        </h1>
        <div
          className="w-48 h-48 flex justify-center items-center spinner-parent"
        >
          <div
            className="rounded-full w-20 h-20 bg-blue-200 spinner"
          />
        </div>
      </div>
    </div>
  </ContentContainer>
</ContentLoading>
`;
